src/blueprints/tv2/action-factories/tv2-transition-effect-action-factory.ts

```diff
@@ -360,26 +360,30 @@ export class Tv2TransitionEffectActionFactory extends ActionFactory {
     return breakerDsk
   }
 
-  private updateTimelineObjectsWithTransitionEffect(action: Tv2TransitionEffectAction, piece: Piece): Tv2TransitionEffectAction {
-    const sourceInput: number | undefined = this.videoMixerTimelineObjectFactory.findProgramSourceInputFromPiece(piece)
-    if (!sourceInput) {
-      this.logger.data({ action, piece }).warn('Can\'t find a Program SourceInput to put the Transition Effect on')
+  private updateTimelineObjectsWithTransitionEffect(action: Tv2TransitionEffectAction, onAirPiece: Piece | undefined, nextPiece: Piece | undefined): Tv2TransitionEffectAction {
+    if (!nextPiece) {
+      return action
+    }
+
+    const nextSourceInput: number | undefined = this.videoMixerTimelineObjectFactory.findProgramSourceInputFromPiece(nextPiece)
+    if (!nextSourceInput) {
+      this.logger.data({ action, nextPiece }).warn('Can\'t find a Program SourceInput to put the Transition Effect on')
```

Comment:
```suggestion
      this.logger.data({ action, nextPiece }).warn(`Can't find a Program SourceInput to put the Transition Effect on`)
```

Reply:
Eslint forces it to be single quotes when not using string interpolation.